### PR TITLE
Visitor API for misc ops

### PR DESCRIPTION
**Pull request by tomdol**

### CODE REVIEW

File: src/ngraph/serializer.cpp [hunk 1]
```diff
@@ -2326,6 +2326,9 @@ shared_ptr<Node> JSONDeserializer::deserialize_node(json node_js)
         case OP_TYPEID::RegionYolo: { break;
         }
         case OP_TYPEID::ReorgYolo: { break;
```
Comment (tomdol) [inline src/ngraph/serializer.cpp:2328]: Not sure if I should have added the serialization/deserialization when implementing the visitor API for this op. What is the current approach?

**Comment by tomdol** [comment]
CI passed in MR#7402

File: src/ngraph/op/roi_pooling.cpp [hunk 2]
```diff
@@ -71,3 +71,11 @@ shared_ptr<Node> op::ROIPooling::clone_with_new_inputs(const OutputVector& new_a
...
         new_args.at(0), new_args.at(1), m_output_size, m_spatial_scale, m_method);
 }
+
+bool op::ROIPooling::visit_attributes(AttributeVisitor& visitor)
+{
+    visitor.on_attribute("output_size", m_output_size);
+    visitor.on_attribute("spatial_scale", m_spatial_scale);
+    visitor.on_attribute("method", m_method);
+    return true;
+}
```
Comment (rkimballn1) [inline src/ngraph/op/roi_pooling.cpp:81]:
```suggestion
}

```

**Comment by tomdol** [comment]
When will the 'dynamic' branch be merged to master? I wanted this PR to go to master directly by following the procedure from an email thread.

**Pull request merged**